src/main/java/pl/lodz/zzpj/kanbanboard/entity/ERole.java

```diff
@@ -0,0 +1,7 @@
+package pl.lodz.zzpj.kanbanboard.entity;
+
+import org.springframework.security.core.GrantedAuthority;
+
+public enum ERole {
```

Comment:
These enum values should be moved as a constant to the Role class. Also, I think the only relevant roles are: USER, ADMIN.